client/knode/api.go

```diff
@@ -256,6 +259,66 @@ func (api *PublicTokenAPI) MintingFinished() (bool, error) {
 	return api.consensus.Token().MintingFinished()
 }
 
+type PendingMintTransaction struct {
+	Id        *big.Int       `json:"id"`
+	To        common.Address `json:"to",omitempty`
+	Amount    *big.Int       `json:"amount",omitempty`
+	Confirmed bool           `json:"confirmed"`
+}
+
+type PendingMintTransactions []PendingMintTransaction
+
+func (api *PublicTokenAPI) MintList() (ret PendingMintTransactions, err error) {
+
+	if err := api.consensus.MintInit(); err != nil {
+		return ret, err
+	}
+
+	multiSig := api.consensus.MultiSigWalletContract()
```

Comment:
MultiSigWallet should not be part of the consensus package

Reply:
let's make an issue for that and address it in a different PR